--- fancy-0.1.0.orig/lib/boot.fnc
+++ fancy-0.1.0/lib/boot.fnc
@@ -7,24 +7,24 @@
 # In general, it's best to add any autoload-files at the end of the
 # current list (if adding them here is really necessary at all).
 
-require: "object";
-require: "class";
-require: "true_class";
-require: "nil_class";
-require: "number";
-require: "enumerable";
-require: "string";
-require: "array";
-require: "block";
-require: "file";
-require: "directory";
-require: "fancy_spec";
-require: "console";
-require: "hash";
-require: "set";
-require: "symbol";
+require: "/usr/lib/fancy/object";
+require: "/usr/lib/fancy/class";
+require: "/usr/lib/fancy/true_class";
+require: "/usr/lib/fancy/nil_class";
+require: "/usr/lib/fancy/number";
+require: "/usr/lib/fancy/enumerable";
+require: "/usr/lib/fancy/string";
+require: "/usr/lib/fancy/array";
+require: "/usr/lib/fancy/block";
+require: "/usr/lib/fancy/file";
+require: "/usr/lib/fancy/directory";
+require: "/usr/lib/fancy/fancy_spec";
+require: "/usr/lib/fancy/console";
+require: "/usr/lib/fancy/hash";
+require: "/usr/lib/fancy/set";
+require: "/usr/lib/fancy/symbol";
 
 # version holds fancy's version number
-require: "version";
+require: "/usr/lib/fancy/version";
 
 
 # handle ARGV special cases